src/gamemode_defense.cpp

```diff
@@ -861,9 +805,9 @@ std::string defense_location_description( defense_location location )
         case DEFLOC_NULL:
             return "NULL Bug.  (defense.cpp:defense_location_description)";
         case DEFLOC_HOSPITAL:
-            return                 _( "One entrance and many rooms.  Some medical supplies." );
+            return                 _( "One entrance and many rooms. Some medical supplies." );
```

Comment:
These double spaces after `.` in sentences are on purpose for monospace text - [there's some debates](https://en.wikipedia.org/wiki/Sentence_spacing) but that's the style cata uses, you'll see them everywhere

Reply:
The string was one character too long. I'll find a different way to shorten it.